src/App.tsx

```diff
@@ -8,7 +8,58 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+
+import { Todo } from './types/Todo';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [filterSelect, setFilterSelect] = useState('all');
+  const [query, setQuery] = useState('');
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [initialTodos, setInitialTodos] = useState<Todo[]>([]);
```

Comment:
what's the difference between todos and initialTodos?

Reply:
I need `initial todos` array for filtering it each time I change filter type or query
and `todos` is already filtered array which users see